### DISCUSSION TITLE
Refactoring

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#360] [General]
During the meeting, Stefan mentioned that whenever you don't have anything to work on, you can look into the codebase and find some part of it that you'd like to improve. You can refactor it, make the code more readable, reduce some repetition, etc.

But before that, first, make sure you really don't have any tasks you can pick up from the Ready column.

@amos-kibet, @kagure-nyakio

**Comment by wintermeyer** [comment]
And please create issues if you find something which cries for refactoring but you don't have time to do it yourself.
